Oqtane.Client/Modules/ModuleBase.cs

```diff
@@ -45,19 +46,13 @@ public string ModulePath()
             return "Modules/" + GetType().Namespace + "/";
         }
 
-        // url methods
-        public string NavigateUrl()
+        public string NavigateUrl(string path = "", string parameters = "")
         {
-            return NavigateUrl(PageState.Page.Path);
-        }
-
-        public string NavigateUrl(string path)
-        {
-            return NavigateUrl(path, "");
-        }
+            if (path == string.Empty && parameters == string.Empty)
+            {
+                path = PageState.Page.Path;
```

Comment:
The use of `PageState.Page.Path` is here, if you notice earlier it was used on the `NavigateUrl()` overload which is the similar things here .. nothing breaking while it uses an optional params